lib/vagrant-service-manager/services/open_shift.rb

```diff
@@ -6,12 +6,14 @@ class OpenShift < ServiceBase
       def initialize(machine, env)
         super(machine, env)
         @service_name = 'openshift'
-        @extra_cmd = build_extra_command
       end
 
       def execute
         if service_start_allowed?
-          command = "#{@extra_cmd} sccli openshift"
+          command = 'sccli openshift'
```

Comment:
Out of interest, why do you use string interpolation here. Would it not be much easier to use plain string concatenation. IMO it would also be a bit clearer.

On a different note, what do you think about renaming _open_shift.rb_ to _openshift.rb_. This underscore keeps nagging me.